File: 2215-find-the-difference-of-two-arrays/2215-find-the-difference-of-two-arrays.py
class Solution:
    def findDifference(self, nums1: List[int], nums2: List[int]) -> List[List[int]]:
        left = []
        right = []
        for i,k in enumerate(nums1):
            if k in nums2:
                continue 
            else:
                left.append(k)
        for i,k in enumerate(nums2):
            if k in nums1:
                continue
            else:
                right.append(k)
        return [list(set(left)),list(set(right))]

File: 2215-find-the-difference-of-two-arrays/2215-find-the-difference-of-two-arrays.rs
impl Solution {
    pub fn find_difference(nums1: Vec<i32>, nums2: Vec<i32>) -> Vec<Vec<i32>> { 
        let mut left:Vec<i32> = vec![];
        let mut right:Vec<i32> = vec![];
        {   for (i,k) in nums1.iter().enumerate(){
                if  !nums2.contains(k){
                    if !left.contains(k){
                        left.push(*k);
                    }
                }
            } 
        }

        for (i,k) in nums2.iter().enumerate(){
            if  !nums1.contains(k){
                if !right.contains(k){
                    right.push(*k);
                }
            }
        } 

        return vec![left,right];
    }
}